Dockerfile

```diff
@@ -2,4 +2,6 @@ FROM yastdevel/cpp-tw
 # the tests require specific locale settings to pass
 ENV LANG=POSIX LC_ALL=
 COPY . /tmp/sources
-
+# Remove the preinstalled yast2-core, it interferes with the built one
+# when running the tests... (huh??)
```

Comment:
Um, actually it builds the RPM package via `rpmbuild`, it's installed into `/usr/src/packages/BUILDROOT/...`. No idea why the system package makes troubles...